lib/providers/services/http_service.dart

```diff
@@ -8,19 +8,29 @@ import 'package:seeds/constants/http_mock_response.dart';
 import 'package:seeds/models/models.dart';
 import 'package:seeds/providers/notifiers/voted_notifier.dart';
 
-class HttpService {
+class HttpService extends ChangeNotifier {
   String baseURL = Config.defaultEndpoint;
   String hyphaURL = Config.hyphaEndpoint;
   String userAccount;
+  String tokenSymbol;
   bool mockResponse;
 
-  void update({String accountName, String nodeEndpoint, bool enableMockResponse = false}) {
-    nodeEndpoint = nodeEndpoint;
+  get tokenContract => tokenSymbol == "SEEDS" ? "token.seeds" : "seedsdiadems";
+
+  void update(
```

Comment:
token symbol should not have to go into all update methods

It should not be stored in more than 1 place. It's an app wide setting.